.gitmodules

```diff
@@ -65,6 +65,10 @@
 	path = Sming/Components/rboot/rboot
 	url = https://github.com/mikee47/rboot
 	ignore = dirty
+[submodule "seriallib"]
```

Comment:
May be it will be better to move serialib to Arch/Host/Components. It is only Host component and it was my mistake making it available for all architectures.